src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -106,8 +107,38 @@
         ANNOTATION_TYPE_MEMBER_REQUIRED,
         PROPERTIES;
 
-        public static final EnumSet<Kind> summarySet = EnumSet.range(INNER_CLASSES, METHODS);
-        public static final EnumSet<Kind> detailSet = EnumSet.range(ENUM_CONSTANTS, METHODS);
+        private static final EnumSet<Kind> defaultSummarySet = EnumSet.of(
+                INNER_CLASSES, FIELDS, CONSTRUCTORS, METHODS);
```

Comment:
Picky, maybe for another time: the correct term is probably `NESTED_CLASSES`.  Inner classes are a subset of nested classes that have an `outer` instance. (i.e. they're not `static` nested classes.